packages/suite/src/views/wallet/send/components/Options/components/BitcoinOptions/components/CoinControl.tsx

```diff
@@ -0,0 +1,218 @@
+    margin-top: 6px;
+`;
+
+const GreyText = styled.div`
+    color: ${({ theme }) => theme.TYPE_LIGHT_GREY};
+`;
+
+const DustDescriptionRow = styled(GreyText)`
+    margin: 6px 0 12px 0;
+`;
+
+const StyledSwitch = styled(Switch)`
+    margin: 0 14px 0 auto;
+`;
+
+const AmountWrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    margin-left: auto;
+    text-align: right;
+`;
+
+const MissingToInput = styled.div<{ isVisible: boolean }>`
+    visibility: ${({ isVisible }) => !isVisible && 'hidden'};
+`;
+
+const StyledPagination = styled(Pagination)`
+    margin-top: 20px;
+`;
+
+const Line = styled.div`
+    width: 100%;
+    height: 1px;
+    border-top: 1px solid ${({ theme }) => theme.STROKE_GREY};
+    margin: 12px 0 16px 0;
+`;
+
+interface Props {
+    close: () => void;
+}
+
+export const CoinControl = ({ close }: Props) => {
+    const [currentPage, setSelectedPage] = useState(1);
+
+    const { addToast, fetchTransactions } = useActions({
+        addToast: notificationActions.addToast,
+        fetchTransactions: fetchTransactionsThunk,
+    });
+
+    const { translationString } = useTranslation();
+
+    const {
+        account,
+        allUtxosSelected,
+        composedInputs,
+        dustUtxos,
+        errors,
+        getDefaultValue,
+        isCoinControl,
+        network,
+        outputs,
+        selectedUtxos,
+        spendableUtxos,
+        toggleCheckAllUtxos,
+        toggleCoinControl,
+    } = useSendFormContext();
+
+    const { areSatsUsed } = useBitcoinAmountUnit(account.symbol);
+
+    const inputs = isCoinControl ? selectedUtxos : composedInputs;
+
+    const getTotal = (amounts: number[]) =>
+        amounts.reduce((previous, current) => previous + current, 0);
+    const getFormattedAmount = (amount: number) =>
+        formatNetworkAmount(amount.toString(), account.symbol);
+
+    // calculate and format amounts
+    const totalInputs = getTotal(inputs.map(input => Number(input.amount)));
+    const totalOutputs = getTotal(
+        outputs.map((_, i) => Number(getDefaultValue(`outputs[${i}].amount`, ''))),
+    );
+    const totalOutputsInSats = areSatsUsed ? totalOutputs : totalOutputs * 10 ** network.decimals;
+    const missingToInput = totalOutputsInSats - totalInputs;
+    const formattedTotal = getFormattedAmount(totalInputs);
+    const formattedMissing = getFormattedAmount(missingToInput);
+    const isMissingToAmount = missingToInput > 0;
+    const isMissingVisible =
+        isCoinControl &&
+        (isMissingToAmount ||
+            !!errors.outputs?.some(
+                error =>
+                    ((error?.amount as TypedFieldError)?.message as ExtendedMessageDescriptor)
+                        ?.id === 'TR_NOT_ENOUGH_SELECTED',
+            ));
+    const missingToInputId = isMissingToAmount ? 'TR_MISSING_TO_INPUT' : 'TR_MISSING_TO_FEE';
+
+    // pagination
+    const totalItems = account.utxo?.length || 0;
+    const utxosPerPage = SETTINGS.TXS_PER_PAGE;
```

Comment:
Maybe, but at the same time it would be hard to select from 100 UTXOs if the count per page is even shorter. 

I do not plan to change it until somebody confirms another value. cc @hynek-jina

Reply:
Currently it is 25?
I don't have better number.. (and if yes, I would think rather about bigger one)
I would leave it as it is.. we can modify that later...